airflow/contrib/hooks/ssh_hook.py

```diff
@@ -56,6 +57,7 @@ def __init__(self,
                  username=None,
                  password=None,
                  key_file=None,
+                 port=SSH_PORT,
```

Comment:
Please update the DocString above with this arg, thanks!

Reply:
: ) Done. Thanks!